Tvl.Collections.Trees.Test/AbstractSetTest.cs

```diff
@@ -307,6 +307,67 @@ public void TestSetEquals()
 
         protected abstract ISet<T> CreateSet<T>();
 
+        protected static void TestICollectionInterfaceImpl(ICollection collection, bool supportsNullValues)
+        {
+            Assert.False(collection.IsSynchronized);
+
+            Assert.IsType<object>(collection.SyncRoot);
+            Assert.Same(collection.SyncRoot, collection.SyncRoot);
+
+            if (supportsNullValues)
+            {
+                var copy = new object[collection.Count];
+
+                Assert.Throws<ArgumentNullException>(() => collection.CopyTo(null, 0));
```

Comment:
:memo: The first three assertions here were previously present in `TreeSetTest` but not in `SortedTreeSetTest`. Now both cases are covered.